tests/test_wikiprocess.py

```diff
@@ -4199,7 +4199,17 @@ def test_get_page_with_namespace_prefixes(self):
         )
         self.assertEqual(self.ctx.get_page("mod:title", 828), module_page)
 
-
+    def test_unnamed_template_arg_end_in_newline(self):
+        # https://ru.wiktionary.org/wiki/adygejski
+        # newline at the end of unnamed template argument should be removed
+        self.ctx.add_page("Template:test", 10, "{{{1}}}")
+        self.ctx.start_page("")
+        self.assertEqual(
+            self.ctx.expand("{{test| \n unnamed1 \n}}"), " \n unnamed1 "
+        )
+        self.assertEqual(
+            self.ctx.expand("{{test| \n {{test|unnamed2}} \n}}"), " \n unnamed2 "
+        )
```

Comment:
This commit changes the template args but not the args used in Lua:

```suggestion
        self.ctx.add_page("Template:test", 10, "{{{1}}}")
        self.ctx.add_page("Template:testlua", 10, "{{#invoke:test|test|{{{1}}}}}")
        self.ctx.add_page(
            "Module:test",
            828,
            """
            local export = {}

            function export.test(frame)
              return frame:getParent().args[1] .. "-" .. frame.args[1]
            end

            return export
            """,)
        self.ctx.start_page("")
        self.assertEqual(
            self.ctx.expand("{{test| \n unnamed1 \n}}"), " \n unnamed1 "
        )
        self.assertEqual(
            self.ctx.expand("{{test| \n {{test|unnamed2}} \n}}"),
            " \n unnamed2 ",
        )
        self.assertEqual(
            self.ctx.expand("{{testlua| \n unnamed3 \n}}"), " \n unnamed3 - \n unnamed3"
        )
```

If the last assert passed then the Lua error in ru edition page should be fixed.